compiler/src/test/scala/org/scalajs/core/compiler/test/JSExportTest.scala

```diff
@@ -11,6 +11,8 @@ class JSExportTest extends DirectTest with TestHelpers {
 
   @Test
   def noDoubleUnderscoreExport = {
+    // scalastyle:off
```

Comment:
If it spills on the next files, I'll just cancel this PR. This would be an irresponsible bug of ScalaStyle that I wouldn't fell comfortable living with. @ngbinh What makes you think ScalaStyle has such a major flaw?